vellum/workflows/triage_message/nodes/triage_message_node.py

```diff
@@ -104,6 +104,11 @@ def job_opportunity_response(
     express genuine interest in the role, and explain how your automated development skills could 
     benefit their team. The forwarder will automatically receive a BCC copy of your response to 
     stay informed.
+    
+    CRITICAL: In your initial paragraph, acknowledge that while this email was originally intended 
+    for the person who forwarded it (David Vargas), that person is not currently available 
```

Comment:
I meant remove David Vargas entirely. My name should never be mentioned in the system prompt